test/uk/gov/hmrc/minorentityidentificationfrontend/services/ValidateTrustKnownFactsServiceSpec.scala

```diff
@@ -23,7 +23,7 @@ import uk.gov.hmrc.http.HeaderCarrier
 import uk.gov.hmrc.minorentityidentificationfrontend.connectors.mocks.MockRetrieveTrustKnownFactsConnector
```

Comment:
I would add a test for the connector returning NOT FOUND.

Reply:
Updated